### PR TITLE
Update exam-policies.md

### PR DESCRIPTION
**Pull request by ines-mmarques**
Added more FAQuestions related to training, this way customers will have easier access to these.

### CODE REVIEW

File: src/community/training/exam-policies.md [hunk 1]
```diff
@@ -7,23 +7,103 @@ app_type: traditional web apps, mobile apps, reactive web apps
...
+# OutSystems certifications FAQs
 
-OutSystems exam candidates accept the reschedule and cancelation terms and conditions during the registration process:
+During the registration process, as an OutSystems exam candidate, you will be prompted to accept the reschedule and cancelation terms and conditions:
 
 ![exam-policies](images/prometric-exam-policies.png)
 
-**The period in which it is possible to reschedule exams and associated fees:**
-To be able to reschedule exams, candidates need to do it up to 15 days before the exam’s original scheduled date and time. Rescheduling the exam 14 days ahead of its date, and up to 48 hours ahead, will incur an extra fee of 25 US$. It's impossible to reschedule the exam less than 48 hours before the date, so be sure to do it, if you wish, before that.
+You can also download them directly from OutSystems Certifications page:
```
Comment (OS-filipaoliveira) [inline src/community/training/exam-policies.md:16]: Is the colon at the end of this sentence a typo?

File: src/community/training/exam-policies.md [hunk 2]
```diff
@@ -7,23 +7,103 @@ app_type: traditional web apps, mobile apps, reactive web apps
...
+### Are the slots are in AM or PM?
+The time format on the scheduling system is 24 hour. This means that if the slots available are, for example, 04:00 or 04:45, it means these slots are AM. 
+If you want to schedule your exam in the afternoon, please make sure the time slots are displayed as 16h00 or 16:45, for example.
+The time format on the scheduling system depends on your Google Chrome Language settings (Settings > Advanced > Languages > Languages section > Click on the drop-down arrow). If it is set for English, for example, the time format on the scheduling system will be in AM/PM. If it is set for Portuguese, for instance, the time will follow the 24 hours format.
+With this in mind, we advise you to first check your Google Chrome settings before scheduling the exam, this way you will prevent any confusion when selecting the exam time. We also remind you that, as shared in the [User Guide](https://www.prometric.com/sites/default/files/2020-04/PrometricProUserGuide_3.1_1.pdf), it is recommended to use Google Chrome when taking the exam.
```
Comment (OS-filipaoliveira) [inline src/community/training/exam-policies.md:26]: I checked the User guide and couldn't find any browser advice information. Is this Google Chrome recommendation up to date?

File: src/community/training/exam-policies.md [hunk 3]
```diff
@@ -7,23 +7,103 @@ app_type: traditional web apps, mobile apps, reactive web apps
...
+## Scheduling
 
-**Where to reschedule or cancel an exam:**
-Candidates with a scheduled exam can reschedule by accessing the [Certification Page](https://www.outsystems.com/learn/certifications/), selecting the scheduled exam and clicking the Reschedule or Cancel button, following the steps.
+### Are the slots are in AM or PM?
+The time format on the scheduling system is 24 hour. This means that if the slots available are, for example, 04:00 or 04:45, it means these slots are AM. 
+If you want to schedule your exam in the afternoon, please make sure the time slots are displayed as 16h00 or 16:45, for example.
+The time format on the scheduling system depends on your Google Chrome Language settings (Settings > Advanced > Languages > Languages section > Click on the drop-down arrow). If it is set for English, for example, the time format on the scheduling system will be in AM/PM. If it is set for Portuguese, for instance, the time will follow the 24 hours format.
```
Comment (OS-filipaoliveira) [inline src/community/training/exam-policies.md:25]: First, we say the time format on the scheduling system is 24-hour, but then we say the time format depends on the browser language settings. Which option is the correct one?

**Comment by ines-mmarques** [comment]
Hello Filipa,

Thank you for reaching back.

It's actually not a typo, however, there should be an image below:
[image: image.png]

The image is a printscreen from the OutSystems Certifications page.

Thank you!

File: src/community/training/exam-policies.md [hunk 4]
```diff
@@ -7,23 +7,103 @@ app_type: traditional web apps, mobile apps, reactive web apps
...
+
+
+### How can I change my exam appointment from online/test center to test center/online appointment? 
+<div class="info" markdown="1">
+
+It's impossible to reschedule the exam less than 48 hours before the exam’s scheduled date. So if you wish, make sure you do it before that.
+
+</div>
+
+To change your exam appointment to an online/test center, you need to visit this page, cancel the appointment, and register again on the Certification Page, select the option “Remote”/“Certification Center”.
```
Comment (OS-filipaoliveira) [inline src/community/training/exam-policies.md:57]: What is "this page"?

File: src/community/training/exam-policies.md [hunk 5]
```diff
@@ -7,23 +7,103 @@ app_type: traditional web apps, mobile apps, reactive web apps
...
+### Where can I find my certificate after I pass my exam?
+Your certificate is available for you to download on your [Community profile](https://www.outsystems.com/community/) account. Learn how in [this guide](https://success.outsystems.com/support/outsystems_community/training_and_certifications_support/how_to_download_your_outsystems_certificate/).
+
+### Can OutSystems review my exam?
+OutSystems does not review any exams. Nonetheless, we make available the exam report you receive with the result, the topics of the questions you failed and we can ensure that exams are correctly assessed and therefore, given scores are accurate and correct.
+
+
+## Other Questions
+
+### Will the Covid-19 Pandemic affect my exam appointment?
```
Comment (OS-filipaoliveira) [inline src/community/training/exam-policies.md:107]: Does this question still apply nowadays?

**Pull request closed without merging**